rustic-lsp.el

```diff
@@ -152,18 +162,13 @@ with `lsp-rust-switch-server'."
       (let ((inhibit-read-only t))
         (erase-buffer)
         ;; wrap expanded macro in a main function so we can run rustfmt
-        (insert "fn main()")
+        (insert "fn main() {")
```

Comment:
I guess they have changed the output.

Reply:
Attribute macroses can expand to one or more statements and cant be formatted with old method

Reply:
I see.